index.html

```diff
@@ -0,0 +1,236 @@
+            <p class="pc-text m-2">Here you will find more information about obtaining tickets to the Inti Raymi Music Festival. </p>
+          </a>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/scheduleicon.svg" alt="schedule icon">
+          <h3><a href="#" class="program-cards-title">Schedule</h3></a>
+          <p class="pc-text m-2">The Festival will take place on June the 22nd, 2023, starting at noon. See inside for more information.</p>
+        </a>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/servicesicon.svg" alt="food and services icon">
+          <h3><a href="#" class="program-cards-title">Food & Services</a></h3>
+          <p>You will have a variety of services available like food and souvenir stands. See inside
+            for more details</p>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/exibitionicon.svg" alt="Exhibition icon">
+          <h3><a href="#" class="program-cards-title">Inti Raymi Exhibition</a></h3>
+          <p>Join the exhibition to learn more about this incredible celebration. It is for all public that attend the
+            festival!</p>
+        </div>
+
+        <div class="program-cards">
+          <img src="./capstoneimgs/programicon.svg" alt="Forum icon">
+          <h3><a href="#" class="program-cards-title">Forum</a></h3>
+          <p>Foreigners and nationals are welcome in this festival! Get in touch with other people interested in this event.</p>
+        </div>
+
+        <button class="join-inti">Join Inti Raymi Celebration 2023</button>
+      </div>
+
+      <div class="desktop-see d-none d-md-flex justify-content-center py-5 mt-5">
+        <h4 href="#" class="desktop-see-program">SEE THE WHOLE PROGRAM</h4>
+      </div>
+
+    </section>
+
+    <div class="perform-cont">
+
+    </div>
+
+    <section class="main-performers" id="performers-list ">
+
+      <div class="d-flex justify-content-center" id="performer-title">
+        <h1>Performers</h1>
+      </div>
+
+      <div class="container-performers p-0 d-flex ">
+        
+        <article id="all-performers" class="d-flex gap-4">
+
+          <!-- <div class='card fl flex-md-row pt-3'>
+
+            <img  src='./capstoneimgs/tawaimg.png' class='card-img-top' 
+              alt='...'>
+            
+            <div class = "band-detail card-body col-12">
+                <h4 class="band-name">tawa</h4>
+                <div class="band-brief-info">Tawa is an ecuadorian band from Quito. Formed in Argentina in 2013.
+                </div>
+                <div class="grey-line"></div>
+                <p class="artist-intro">Focusing in latinamerican rhythms and folklore, TAWAs artistic persuit has led them to represent Ecuador in international festivals playing traditional music with a modern approach.-->
+          
+
+        </article>
+      </div>
+    </section>
```

Comment:
- [ ] Design-wise, please consider reducing the size of elements in your featured speaker's section and include the orange divider under the title, it is important to follow the design guidelines! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/226499108-a001370a-f773-4561-bbdc-7d311be5c137.png) | ![image](https://user-images.githubusercontent.com/77069011/226499059-8586cd61-3a71-4869-8a53-135c298db9c2.png) |

- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧

Reply:
Thank you, it is done!

Reply:
- [ ] This issue is not yet well-resolved :x: 

- [ ] According to the previous reviewer's suggested changes on reducing the sizes of your element, the content in this section still has too big a font size and the text content is too much. You should play around with the font sizes by reducing them to match at least, the same size or even not the same size with the font size from the design. 
- [ ] You can reduce the text content so they are not much. Your text is too long in length. try reducing them. compare your text length to what you see on the design. Are they close? of course they're not. let's fix this to improve the design.



*compare your one card with that of the original design then fix the differences you can see*
![Screenshot from 2023-03-21 17-00-04](https://user-images.githubusercontent.com/59397159/226670052-fdfa42bf-7685-455e-82d8-a44a6072b5b5.png)